content/courses/program-security/signer-auth.md

```diff
@@ -339,7 +339,9 @@ pub mod signer_authorization {
 #[derive(Accounts)]
 pub struct InitializeVault<'info> {
     #[account(
-        init,
+        // We use init_if_needed here for the test. Otherwise, the test will result in an error "Already in use" after testing once.
+        // Use "init" if you want to ensure that the "initialize_vault" function runs only once.
+        init_if_needed,
```

Comment:
See comments in your PR to the other repo.

Reply:
see https://github.com/Unboxed-Software/solana-signer-auth/pull/3#discussion_r1735474018